docs/source/commands.rst

```diff
@@ -118,4 +118,21 @@ Re-evaluates your Mergify rules on this pull request.
   * - Example
     - ``@Mergifyio refresh``
 
+unqueue
```

Comment:
```suggestion
.. _unqueue command:

unqueue
```
for other commands, the ref `.. _action command:` is before the title right ? 

it also seems like the squash command is missing its ref too `.. _squash command:`

Reply:
I removed it.

".. _ref" is useful to create anchor or RST reference. We don't really need it as "unqueue" is a title and a title always have an anchor already. We don't need the ref either as it's used nowhere.